repository/src/main/java/org/alfresco/repo/jscript/RhinoScriptProcessor.java

```diff
@@ -108,7 +109,24 @@ public class RhinoScriptProcessor extends BaseProcessor implements ScriptProcess
     /** Cache of runtime compiled script instances */
     private final Map<String, Script> scriptCache = new ConcurrentHashMap<String, Script>(256);
     
-    
+    /** Rhino optimization level */
+    private int optimizationLevel = -1;
+
+    /** Maximum seconds a script is allowed to run */
+    private int maxScriptExecutionSeconds = -1;
+
+    /** Maximum of call stack depth (in terms of number of call frames) */
+    private int maxStackDepth = -1;
+
+    /** Maximum memory (bytes) a script can use */
+    private long maxMemoryUsed = -1L;
```

Comment:
Changing the name to `maxMemoryUsedInBytes` is more convenient and less error prone without reading the comment!

Reply:
Added `InBytes` suffix to every occurrence of `maxMemoryUsed` property.